### DISCUSSION TITLE
Scaricamento flusso per intervallo data

### DISCUSSION THREAD

**Discussion by donlegro** [fattureincloud/api#95] [Q&A] [answered]
Salve nei filtri delle chiamate received_documents e received_documents e issued_documents non riesco a capire se si può inserire un intervallo data.
Posso inserire una data precisa ma non un intervallo. Per aggiornare quindi sono costretto a scaricare tutto e poi scorrere le pagine e prendere quello che mi serve.
Grazie in anticipo.

**Comment by valmoz** [comment, marked as answer]
Buongiorno @donlegro 
dovrebbe essere sufficiente utilizzare gli operatori  di comparazione '<', '<=', '>', '>=' sul campo che le serve filtrare.

Ci faccia sapere se in questo modo è in grado di filtrare la richiesta.

Grazie
Mauro

**Comment by donlegro** [comment]
Sì grazie alla fine scavando nei filtri sono riuscito a trovare la soluzione. Bisogna valorizzare il parametro q con una stringa del genere date>='2022-01-01' and date<='2022-01-31'
